### PR TITLE
Add more detail on how to set up jdk

### PR DESCRIPTION
**Pull request by lunaleaps**
Add more detail to macOS steps for installing JDK. 

CleanShot 2023-04-06 at 17 54 49@2x

### CODE REVIEW

File: docs/_getting-started-macos-android.md [hunk 1]
```diff
@@ -26,8 +26,13 @@ We recommend installing the OpenJDK distribution called Azul **Zulu** using [Hom
 ```shell
 brew tap homebrew/cask-versions
 brew install --cask zulu11
+
+# Get path to where cask was installed to double-click installer
+brew info --cask zulu11
 ```
 
+After you install the JDK, update your `JAVA_HOME` environment variable. If you used above steps, JDK will likely be at `/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home`
```
Comment (cortinico) [inline docs/_getting-started-macos-android.md:34]: You should not need to manipulate the `JAVA_HOME` environment variable as Brew does that. Why do we ask the user to do so?

Comment (lunaleaps) [reply docs/_getting-started-macos-android.md:34]: Hmm I tried this again, and following the brew cask install and the installer did not update my `JAVA_HOME`.

It looks like this [cask code](https://github.com/Homebrew/homebrew-cask/blob/a93e2a47d3d1866ac5f222854eedfd627e60ab5d/Casks/zulu.rb) doesn't set that up, I couldn't find any documentation on Azul's page about whether they set this up: https://www.azul.com/downloads/?package=jdk#zulu

Comment (cortinico) [reply docs/_getting-started-macos-android.md:34]: Sorry I missed this comment. I belive you're right. I was using `jenv` that's why I haven't noticed this.

**Review by cortinico — APPROVED** [review]
We can merge this @lunaleaps sorry for the late review

**Pull request merged**